wifite/config.py

```diff
@@ -180,15 +180,15 @@ def parse_settings_args(cls, args):
             Color.pl('{+} {C}option:{W} targeting BSSID ' +
                     '{G}%s{W}' % args.target_bssid)
 
-        if args.five_ghz == True:
+        if args.five_ghz:
             cls.five_ghz = True
             Color.pl('{+} {C}option:{W} including {G}5Ghz networks{W} in scans')
 
-        if args.show_bssids == True:
+        if args.show_bssidS:
```

Comment:
This should be "if args.show_bssids:" with small "s" at the end - don't know why it come with capital letter from my code..